src/sentry/grouping/fingerprinting/__init__.py

```diff
@@ -175,8 +193,10 @@ def _from_config_structure(cls, data):
             raise ValueError("Unknown version")
         return cls(rules=[Rule._from_config_structure(x) for x in data["rules"]], version=version)
 
-    def _to_config_structure(self):
-        return {"version": self.version, "rules": [x._to_config_structure() for x in self.rules]}
+    def _to_config_structure(self, include_builtin=False):
+        rules = self.iter_rules() if include_builtin else self.rules
```

Comment:
Should we just be passing `include_builtin` to `iter_rules` instead of having this if statement?